src/osd/CMakeLists.txt

```diff
@@ -45,8 +45,15 @@ add_library(osd STATIC ${osd_srcs}
   $<TARGET_OBJECTS:global_common_objs>
   $<TARGET_OBJECTS:heap_profiler_objs>)
 target_link_libraries(osd ${LEVELDB_LIBRARIES} dmclock ${CMAKE_DL_LIBS} ${ALLOC_LIBS})
+target_link_libraries(osd lttng-ctl)
+if(WITH_LTTNG_LOGGING)
+  add_dependencies(osd bluestore-parsetool)
+  target_link_libraries(osd logging_tp)
+  target_link_libraries(osd lttng-ust-fork)
```

Comment:
might want to add dependencies from `os` to `*_impl.h`, as `BlueStore.cc` now includes them. could use `set_source_files_properties()` for this purpose.

Reply:
Noted, thanks.